django/contrib/admin/filters.py

```diff
@@ -220,7 +221,7 @@ def choices(self, cl):
                 'query_string': cl.get_query_string({
                     self.lookup_kwarg_isnull: 'True',
                 }, [self.lookup_kwarg]),
-                'display': EMPTY_CHANGELIST_VALUE,
+                'display': mark_safe(self.empty_value_display),
```

Comment:
I rather have the `mark_safe()` inside `model_admin.get_empty_value_display()` and call that method here explicitly instead of storing its value in `__init__`. This prevents limits XSS vectors when people override `get_empty_value_display()` and return unescaped content.